### DISCUSSION TITLE
NEB calculations

### DISCUSSION THREAD

**Discussion by Computational-Nanoscience-group-USF** [deepmodeling/abacus-develop#1827] [Q&A] [answered]
Hello,
Can I perform NEB calculations using ABACUS?

thank you.

**Comment by hongriTianqi** [comment, marked as answer]
Hi, thanks for you question! NEB algorithms have not been implemented in ABACUS code. However, you can use our [ABACUS-ASE interface](https://gitlab.com/1041176461/ase-abacus) to perform NEB calculations by calling the ABACUS calculator. If you have further questions, welcome to contact us.

**Comment by QuantumMisaka** [comment]
NEB and CI-NEB calculation function using ASE-ABACUS interface is under development, you can pay attention to https://github.com/QuantumMisaka/ase-neb-abacus
